fdbrpc/Stats.h

```diff
@@ -136,7 +138,16 @@ struct SpecialCounter final : ICounter, FastAllocated<SpecialCounter<F>>, NonCop
 	void remove() override { delete this; }
 
 	std::string const& getName() const override { return name; }
-	int64_t getValue() const override { return f(); }
+	int64_t getValue() const override {
+		auto result = f();
+		if constexpr (std::is_floating_point_v<decltype(result)>) {
```

Comment:
There have been cases where people tried to publish a float here not realizing that it would be truncated to an integer. Would it be crazy to disallow using floats rather than converting them?

Reply:
I think PercentageOfDefaultGRVQueueProcessed is one such instance. Let met try disallowing and see how much of a change that is

Reply:
> There have been cases where people tried to publish a float here not realizing that it would be truncated to an integer. Would it be crazy to disallow using floats rather than converting them? I don't know how often we are intentionally passing floats.

Suggestion implemented. It looks good IMO